exercises/etl/instructions.md

```diff
@@ -0,0 +1,22 @@
+# Instructions
+
+Your task is to change the data format of letters and their point values in the game.
+
+Currently, letters are stored as a list, per score.
+
+- 1 point: "A", "E", "I", "O", "U", "L", "N", "R", "S", "T",
+- 2 points: "D", "G",
+- 3 points: "B", "C", "M", "P",
+- 4 points: "F", "H", "V", "W", "Y",
+- 5 points: "K",
+- 8 points: "J", "X",
+- 10 points: "Q", "Z",
+
+This needs to be changed to store each letter with its score.
+As part of this change, the team has also decided to change the letters to be lower-case rather than upper-case.
+
+- "a" is worth 1 point.
+- "b" is worth 3 points.
+- "c" is worth 3 points.
+- "d" is worth 2 points.
+- etc.
```

Comment:
This is the same input that it used to be; not really sure how to change this without using actual programming syntax... and the syntax varies by language.

Reply:
Ok, I've tried to add some wording to clarify, and added a note to tell people to look at the test suite, since we can't specify an actual code sample in the instructions.